composables/useTransaction.ts

```diff
@@ -177,6 +179,11 @@ export const executeAction = ({
       execBurnMultiple(item as ActionBurnMultipleNFTs, api, executeTransaction),
   }
 
+  if (!isActionValid(item)) {
+    consola.warn(`Invalid action: ${JSON.stringify(item)}`)
+    return 'INVALID ACTION'
+  }
```

Comment:
What if we start to use nuxt error handling?
```suggestion
  if (!isActionValid(item)) {
    consola.warn(`Invalid action: ${JSON.stringify(item)}`)
    throw createError({
      statusCode: 404,
      statusMessage: 'Interaction Not Found'
    })
  }
```